lesson_18/Oleksandr Piddubnyi.py

```diff
@@ -0,0 +1,29 @@
+import sqlite3
+
+with sqlite3.connect('library.db') as connection:
+    cursor = connection.cursor()
+    connection.execute("PRAGMA foreign_keys = ON;")
+
+    cursor.execute('DROP TABLE IF EXISTS Cars')
+
+cursor.execute('''
```

Comment:
your indentation seems off... yes, it will work (because WITH context manager does not handle the lifetime of the database connection, but the transactions), but it is better not to confuse it.